Scenes/Mine/Mine.gd

```diff
@@ -10,8 +10,8 @@ func _on_Mine_body_entered(body):
 	explode()
 
 func cause_damage(victim):
-	if victim and victim.has_method("damage"):
-		victim.damage(10)
+	if victim and victim.has_method("take_damage"):
```

Comment:
changed to take_damage, we just had two ways of dealing damage to things, so unified them